.travis.yml

```diff
@@ -0,0 +1,25 @@
+language: android
+
+jdk:
+  - oraclejdk8
+
+install:
+  - echo y | android update sdk -u -a -t tools
```

Comment:
Taken from: https://github.com/travis-ci/travis-ci/issues/6555#issuecomment-282635361. Travis has had issues with it's "licensing" issues.